match2/commons/match_group_display_matchlist.lua

```diff
@@ -148,8 +148,8 @@ function MatchlistDisplay.Match(props)
 	-- Compute widths of the 2 opponent and 2 score columns. The small offsets
 	-- are due to border splitting from table-layout: auto.
 	local minScoreWidth = 30
-	local scoreWidth = math.max(minScoreWidth, math.floor(0.1 * props.width) - 1)
-	local opponentWidth = 0.5 * (props.width - 5 - 2 * scoreWidth)
+	local scoreWidth = math.max(minScoreWidth, math.floor(0.1 * props.width))
+	local opponentWidth = 0.5 * (props.width - 2 - 2 * scoreWidth)
```

Comment:
```suggestion
	local opponentWidth = 0.5 * (props.width - 2 - (2 * scoreWidth))
```

Everywhere else too, and would be nice if there was a comment above this explaining why these numbers

Reply:
resolved via #76